xtask/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "xtask"
+edition = "2021"
+publish = false
```

Comment:
Could you include meta data: https://github.com/awslabs/duvet/blob/main/duvet/Cargo.toml#L1-L13

Reply:
since it's an internal crate it really doesn't need it